lib/compilers/rust.ts

```diff
@@ -52,11 +55,54 @@ export class RustCompiler extends BaseCompiler {
         this.linker = this.compilerProps('linker');
     }
 
-    getSharedLibraryPathsAsArguments() {
+    override getSharedLibraryPathsAsArguments(libraries, libDownloadPath) {
         return [];
     }
 
-    optionsForBackend(backendOptions, outputFilename) {
+    override getSharedLibraryLinks(libraries): string[] {
+        return [];
+    }
+
+    override getIncludeArguments(libraries) {
+        const includeFlag = '--extern';
+        return _.flatten(
+            _.map(libraries, selectedLib => {
+                const foundVersion = this.findLibVersion(selectedLib);
+                if (!foundVersion) return false;
+                const list: string[] = [];
+                for (const rlib of foundVersion.path) {
+                    list.push(includeFlag, `${foundVersion.name}=${foundVersion.name}/build/debug/${rlib}`);
```

Comment:
Having this path preformatted probably takes away the possibility of local users being able to add libraries, is that something we should account for?